dbSearcher/template.php

```diff
@@ -9,7 +9,8 @@
 		<div class="clearfix">
 		<div class="strict float-r">
       <label for="dbs-strict">Find exact match</label>
-			<input type="checkbox" name="dbs-strict" id="dbs-strict" value="TRUE" />	
+      <?php $checkbox_attributes = $strict? 'checked': '' ?>
+      <input type="checkbox" name="dbs-strict" id="dbs-strict" value="TRUE" <?php echo $checkbox_attributes; ?>/>	
```

Comment:
Nice thought! but this should be in a session just like query and database. See index.php

Reply:
Ok. I can do this.